code/src/components/AmountOf.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const AmountOf = () => {
+  const todos = useSelector(state => state.todos.list.todos)
+  const leftTodo = todos.filter(todo => todo.complete === false)
```

Comment:
if you're just interested in the length of this array you could store that instead of the whole array. You could also check if todo.complete is true or false like this:
`const leftTodo = todos.filter(todo => !todo.complete).length`